src/order/filter/archimedean.lean

```diff
@@ -135,3 +140,106 @@ begin
   ... = (max b 0 * n) * r : by rw [mul_assoc]
   ... ≤ f x * r : mul_le_mul_of_nonneg_right hx (le_of_lt hr)
 end
+
+end linear_ordered_semiring
+
+section linear_ordered_ring
+
+variables [linear_ordered_ring R] [archimedean R]
+
+/-- See also `filter.tendsto.at_top_mul_neg_const` for a version of this lemma for
+`linear_ordered_field`s which does not require the `archimedean` assumption. -/
+lemma tendsto.at_top_mul_neg_const' (hr : r < 0) (hf : tendsto f l at_top) :
+  tendsto (λx, f x * r) l at_bot :=
+begin
+  have h : (λ x, f x * -r) = -λ x, f x * r, { ext, simp, },
+  rw tendsto_at_bot_iff_tends_to_neg_at_top,
+  exact h ▸ tendsto.at_top_mul_const' (neg_pos.mpr hr) hf,
+end
+
+/-- See also `filter.tendsto.at_bot_mul_const` for a version of this lemma for
+`linear_ordered_field`s which does not require the `archimedean` assumption. -/
+lemma tendsto.at_bot_mul_const' (hr : 0 < r) (hf : tendsto f l at_bot) :
```

Comment:
Should this mention that the const is positive?

Reply:
Again agreed but again I'm following existing naming, in this case [https://leanprover-community.github.io/mathlib_docs/order/filter/at_top_bot.html#filter.tendsto.at_bot_mul_const](filter.tendsto.at_bot_mul_const).